parlai/agents/bert_ranker/bert_dictionary.py

```diff
@@ -22,6 +23,7 @@ class BertDictionaryAgent(DictionaryAgent):
     def __init__(self, opt):
         super().__init__(opt)
         # initialize from voab path
+        download(opt['datapath'])
```

Comment:
why do you need to download here?

Reply:
So I think building the dictionary is the very first thing done in a training, which means that if you do not have the data downloaded it fails to load the tokenizer.